conda_smithy/configure_feedstock.py

```diff
@@ -784,6 +784,8 @@ def _azure_specific_setup(jinja_env, forge_config, forge_dir, platform):
     }
     template_files = platform_templates.get(platform, [])
 
+    forge_config['build_setup'] = 'run_conda_forge_build_setup'
```

Comment:
This is configurable on other CI.

Reply:
Since this only matters for conda-forge-ci-setup and is platform dependent in *all* the other CI making this work would require substantial refactors to conda-smithy that is not worth doing right now IMO.

Reply:
Looking at the scripts in `conda-forge-ci-setup` we don't actually need to source them.  The OSX ones modofy the CONFIG adding in the build root.

Reply:
env variables are not set if you don't source. https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/branch2.0/recipe/run_conda_forge_build_setup_osx#L5

Reply:
they don't persist in any case past their script block on azure

Reply:
Any workarounds?

Reply:
On windows, packages like boost with long paths can't be built if we don't have https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/branch2.0/recipe/run_conda_forge_build_setup_win.bat#L15

Reply:
So the best workaround is https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/branch2.0/recipe/run_conda_forge_build_setup_osx#L33-L39